frontend/src/components/RecipientsCard/RecipientsForm.tsx

```diff
@@ -1,28 +1,52 @@
 import React, { useEffect, useState } from 'react';
 import './RecipientsForm.css';
-import { useAppSelector, useAppDispatch } from '../../app/hooks';
 import { toast } from 'react-toastify';
 
+/* Material UI import for dropdown menus */
+import Box from '@mui/material/Box';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
 import {
   getRecipients,
   createRecipient,
   updateRecipient,
   deleteRecipient
 } from '../../features/recipients/recipientsSlice';
-
+import { useAppSelector, useAppDispatch } from '../../app/hooks';
 
 /* Form to add recipient or donor */
-const RecipientsForm = (props: any) => {
-
+function RecipientsForm(props: any) {
   /* Recipient and donor state data here */
-  const [entityType, setEntityType] = useState(props.isUpdate ? props.recipient.EntityType : '');
-  const [areaName, setAreaName] = useState(props.isUpdate ? props.recipient.CombinedAreaName : '');
-  const [recipientName, setRecipientName] = useState(props.isUpdate ? props.recipient.name : '');
-  const [demographicName, setDemographicName] = useState(props.isUpdate ? props.recipient.DemographicName : '');
+  const [entityType, setOrgStruc] = useState(
```

Comment:
To prevent some confusion down the line
```suggestion
  const [orgStruc, setOrgStruc] = useState(
```